src/ducks/HistoricalBalance/BalanceView.js

```diff
@@ -40,6 +41,14 @@ class BalanceView extends React.Component {
             This tool is beta. We give no guarantee data is correct
           </div>
         </div>
+
+        {assets && assets.length === 1 && (
+          <div>
+            <SignalMasterModalForm canRedirect={false} asset={assets[0]} />
+            <br />
```

Comment:
@GarageInc Please, explain why the `br` is necessary here. Isn't the same effect could be achieved with a css?

Reply:
Will remove